filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobPathBuilderSpec.scala

```diff
@@ -91,4 +91,17 @@ class BlobPathBuilderSpec extends AnyFlatSpec with Matchers with MockSugar {
     val fileText = (is.readAllBytes.map(_.toChar)).mkString
     fileText should include ("This is my test file!!!! Did it work?")
   }
+
+  ignore should "resolve a path without duplicating container name" in {
+    val endpoint = BlobPathBuilderSpec.buildEndpoint("coaexternalstorage")
+    val store = BlobContainerName("inputs")
+    val blobTokenGenerator = NativeBlobTokenGenerator(store, endpoint)
+    val fsm: BlobFileSystemManager = BlobFileSystemManager(store, endpoint, 10, blobTokenGenerator)
+
+    val rootString = List(endpoint.value, store.value, "cromwell-execution").mkString("/")
```

Comment:
```suggestion
    val rootString = s"${endpoint.value}/${store.value}/cromwell-execution"
```
This reads more straightforward to me, and gives me more confidence that `rootString` is a `String` than the `String` suffix alone.